components/brave_wallet/browser/eth_data_builder.h

```diff
@@ -45,6 +45,22 @@ bool OwnerOf(uint256_t token_id, std::string* data);
 
 }  // namespace erc721
 
+namespace erc1155 {
+
+// Transfers the ownership of an NFT from one address to another address
```

Comment:
Doesn't this API support both fungible and non-fungible token?

Reply:
period at the end

Reply:
> Doesn't this API support both fungible and non-fungible token?

Yes it does, I updated to 'token'.

> period at the end

I added a period and changed 'Transfers' -> 'Transfer' to make it a complete sentence.